ci.sh

```diff
@@ -37,25 +37,29 @@ python -c "import sys, struct, ssl; print('python:', sys.version); print('versio
 echo "::endgroup::"
 
 echo "::group::Install dependencies"
-python -m pip install -U pip build
+python -m pip install -U pip uv -c test-requirements.txt
 python -m pip --version
+python -m uv --version
+
+python -m uv pip install build
 
 python -m build
-python -m pip install dist/*.whl
+wheel_package=$(ls dist/*.whl)
+python -m uv pip install "trio @ $wheel_package"
```

Comment:
We can probably lead the path with ./ to get this to work like before: https://github.com/astral-sh/uv/issues/6052

Reply:
Is there a particular reason not to leave it at my current solution?

Reply:
Personal preference TBH. Maybe you disagree but I prefer moving logic out of shell scripts and into program arguments.

Feel free not to do it though.